protobuf/protos/vehicle/state.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
```

Comment:
@dbacchet are you intending to merge these particular protobuf definitions in this PR?

Reply:
yes, they're just some samples without any particular meaning/value